libraries/helper.rb

```diff
@@ -11,6 +11,9 @@ def raise_not_found
 
     def sensitive_default
       new_resource.sensitive = true unless property_is_set?(:sensitive)
```

Comment:
I think this can be 
```ruby
val = new_resource.sensitive ? new_resource.sensitive : true
```
This looks like it's always going to set sensitive as true in Chef 12

Reply:
Setting sensitive to true for chef 12 seemed like my best option.  With Chef 13 and up folks can specify the sensitive property and it works correctly.  

For chef 12 new_resource.sensitive defaults to false instead of nil.  So I can't tell if it has been set and property_is_set?(:sensitive) throws an ArgumentError. The line resources used to always act like sensitive was true before we used the internal file resource so I'm trying go back to that behavior just for chef 12. 

Isn't "val" in the code above is always going to be truthy?

thanks,
   Mark